tests/async/test_vs_claimable.py

```
@@ -0,0 +1,27 @@
+"""Tests for the 'Fight Health Insurance vs Claimable' comparison page."""
+
+from django.test import Client, TestCase
+from django.urls import reverse
+
+
+class TestVsClaimableView(TestCase):
+    """The comparison page should render as a simple static page."""
+
+    fixtures = ["fighthealthinsurance/fixtures/initial.yaml"]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)